test/verify/check-system-services

```diff
@@ -329,6 +333,15 @@ Unit=test.service
 
         b.go("#/")
 
+        # Check that listen property is shown for socket units
+        self.pick_tab(3)
+        b.click(self.svc_sel("dbus.socket"))
+        b.wait_visible("#listen")
+        b.click(".pf-c-breadcrumb a:contains('Services')")
```

Comment:
... and does not show Memory

Reply:
Socket units also have CurrentMemory shown in the UI

Reply:
Oh wow -- that shouldn't ever be something else than "trivial" :grin:

Reply:
Indeed. I could also hide it, but systemctl status shows it as well, so..